packages/documentation/components/ShowCase.vue

```diff
@@ -42,12 +42,12 @@ export default {
 		padding: 0.2rem 0.5rem;
 		font-size: 12px;
 		color: #fff;
-		background: rgba(0, 0, 0, 0.4);
+		background: rgba(0 0 0 / 40%);
```

Comment:
Isn't the new standard `rgb(0 0 0 / 40%)`? or are `rgb` and `rgba` interchangeable when using forward slash?

Reply:
Yes, `rgba` is an alias of `rgb`, [see here](https://www.w3.org/TR/css-color-4/#color-type)

> [rgb()](https://www.w3.org/TR/css-color-4/#funcdef-rgb) and its [rgba()](https://www.w3.org/TR/css-color-4/#funcdef-rgba) alias, which (like the [hex color notation](https://www.w3.org/TR/css-color-4/#hex-color)) specify sRGB colors directly by their red/green/blue/alpha chanels.